framework/auth/oauth_scopes.py

```diff
@@ -124,8 +128,8 @@ class ComposedScopes(object):
     NODE_ALL_WRITE = NODE_ALL_READ + NODE_METADATA_WRITE + NODE_DATA_WRITE + NODE_ACCESS_WRITE
 
     # Full permissions: all routes intended to be exposed to third party API users
-    FULL_READ = NODE_ALL_READ + USERS_READ + ORGANIZER_READ + GUIDS_READ + (CoreScopes.INSTITUTION_READ, )
-    FULL_WRITE = NODE_ALL_WRITE + USERS_WRITE + ORGANIZER_WRITE + GUIDS_READ
+    FULL_READ = NODE_ALL_READ + USERS_READ + ORGANIZER_READ + GUIDS_READ + (CoreScopes.INSTITUTION_READ, CoreScopes.ADDONS_READ, )
+    FULL_WRITE = NODE_ALL_WRITE + USERS_WRITE + ORGANIZER_WRITE + GUIDS_READ + (CoreScopes.NODE_ADDON_WRITE, )
```

Comment:
Under what circumstances would we want a third party to be able to reconfigure an addon on behalf of a user

In particular, because the tokens we collect are issues on behalf of the entire account (not a folder), it seems that this would let a third party effectively browse the entire contents of the user's account, beyond the folders they intended to make visible.

See doc comment on route capabilities here: https://github.com/CenterForOpenScience/osf.io/pull/5532/files#diff-51eef6ea2a6c35e2b080c9a6fe591e38R1589

Based on flowdock discussion, I'll refer this to @brianjgeiger to revisit the use case, and see if we can't work out any way to mitigate this.